spec/app/rails_6.0/app/assets/config/manifest.js

```diff
@@ -0,0 +1,2 @@
+//= link_tree ../images
```

Comment:
Can we generate a bare minimum rails app? The API one indeally

Reply:
Sure, done.